backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/vdsbroker/VdsBrokerObjectsBuilder.java

```diff
@@ -1408,10 +1406,19 @@ public void updateVDSStatisticsData(VDS vds, Map<String, Object> struct) {
 
         vds.setBootTime(assignLongValue(struct, VdsProperties.bootTime));
 
+        updateMemoryUsage(vds, struct);
```

Comment:
I would leave a comment this function should be called after setting the memory/hugepages data (order matters).

Reply:
OK